src/scene_battle_rpg2k3.cpp

```diff
@@ -837,7 +837,15 @@ bool Scene_Battle_Rpg2k3::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBas
 
 	if (play_reflect_anim) {
 		play_reflect_anim = false;
-		action->PlayAnimation();
+		if (Game_System::GetInvertAnimations()) {
+			invert_animation = false;
+			if (action->OriginalTargetsSet()) {
+				invert_animation = (action->GetOriginalTarget()->IsDirectionFlipped() ^ (action->GetOriginalTarget()->GetType() == Game_Battler::Type_Enemy));
```

Comment:
Put this xor logic behind an inline function so it's not copy pasted. Give it a meaningful name so we know what it's trying to do without needing comments.